src/components/SpecialResolution/Resolution.vue

```diff
@@ -54,15 +54,15 @@
                   text
                   color="primary"
                   class="undo-action"
-                  @click="undoSpecialResolutionStore(); isEditing = false"
+                  @click="undoSpecialResolutionStore();"
                 >
                   <v-icon small>
                     mdi-undo
                   </v-icon>
                   <span>Undo</span>
                 </v-btn>
                 <v-btn
-                  v-if="!isEditing && !hasChanged && isCoopCorrectionFiling"
+                  v-if="(!hasChanged || !isCoopCorrectionFiling) && !isEditing"
```

Comment:
Could you add a comment above this button explaining what you _are_ looking for here? (Too many "nots" to easily understand.)

Reply:
Awesome, thanks. This will really help the future developer in this code.